ds3-cli-certification/src/main/java/com/spectralogic/ds3cli/certification/CertificationUtil.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2017 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3cli.certification;
+
+import com.google.common.collect.Lists;
+import com.spectralogic.ds3client.Ds3Client;
+import com.spectralogic.ds3client.commands.spectrads3.*;
+import com.spectralogic.ds3client.helpers.Ds3ClientHelpers;
+import com.spectralogic.ds3client.models.JobStatus;
+import com.spectralogic.ds3client.models.Priority;
+import com.spectralogic.ds3client.models.Quiesced;
+import com.spectralogic.ds3client.models.SpectraUser;
+import com.spectralogic.ds3client.models.bulk.Ds3Object;
+import com.spectralogic.ds3client.networking.FailedRequestException;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.UUID;
+
+import static java.lang.Thread.sleep;
+
+
+public class CertificationUtil {
+
+    private static final ch.qos.logback.classic.Logger LOG =  (ch.qos.logback.classic.Logger) LoggerFactory.getLogger(CertificationUtil.class);
+
+    public static Path createTempFiles(
+           final String prefix,
+           final int numFiles,
+           final long length) throws IOException {
+        LOG.info("Creating {} files of size {}...", numFiles, length);
+        final Path tempDir = Files.createTempDirectory(prefix);
+        for(int fileNum = 0; fileNum < numFiles; fileNum++) {
+            final File tempFile = new File(tempDir.toString(), prefix + "_" + fileNum);
+            final RandomAccessFile raf = new RandomAccessFile(tempFile, "rw");
```

Comment:
Use a try with resource here.  That will make sure that the close is called even if an exception is thrown before the close is called.